pkg/server/server.go

```diff
@@ -157,9 +158,15 @@ func (host *yamlLanguageHost) Run(ctx context.Context, req *pulumirpc.RunRequest
 			return err
 		}
 		defer loader.Close()
-
+		conf := make(map[string]string)
```

Comment:
We may as well save the allocation.
```suggestion
		conf := make(map[string]string, len(req.GetConfig())
```